sdk/core/azure-core/src/samples/java/com/azure/core/util/metrics/MicrometerMetricsOptions.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util.metrics;
```

Comment:
is `azure-core` right place for samples that involve micrometer?

Reply:
this is an example of implementation for API that lives in azure-core. I'm happy to move it somewhere else, but the only other reasonable place would be out of this repo and in azure-samples, where it'd be less discoverable. I'd expect some customers would be interested in it and having it handy and up-to-date would be nice until at least it's implemented in one of the spring packages.

Reply:
Point taken, I'll create a sample app in https://github.com/Azure-Samples/azure-spring-boot-sample and we'll evolve it with the spring team if/once they adopt micrometer impl. 

I don't think this PR is blocked on it since we know Micrometer impl is possible/reasonable, so I'll take sample as a follow-up and remove micrometer from azure-core samples.

Reply:
Mvn me. I didn't notice that dependency is gone from pom. We're good.